internal/amplitude/publish.go

```diff
@@ -0,0 +1,58 @@
+package amplitude
+
+import (
+	"bytes"
+	"net/http"
+	"time"
+
+	"github.com/cockroachdb/errors"
+	"github.com/inconshreveable/log15"
+)
+
+var AMPLITUDE_API_URL = "https://api2.amplitude.com/2/httpapi"
+
+func Publish(amplitudeAPIToken string, jsonReq []byte) error {
+	data := bytes.NewBuffer(jsonReq)
+	req, err := http.NewRequest("POST", AMPLITUDE_API_URL, data)
+	if err != nil {
+		log15.Warn("Could not log Amplitude event", "err", err)
```

Comment:
Generally, I'd suggest having some kind of prefix to help locate the log faster. And this is a valid error (i.e. we must do something), not a warning (i.e. we should keep an eye on).

```suggestion
		log15.Error("amplitude: cannot create new request", "error", err)
```

Reply:
This seems not addressed.